database/migrations/2025_01_07_000001_update_triggers_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('triggers', function (Blueprint $table) {
+            $table->foreign('parent_id')
+                ->references('id')
+                ->on('triggers')
+                ->nullOnDelete();
+
+            $table->index('parent_id');
+        });
```

Comment:
Since you're updating this table anyway, now might be a good time to drop the `stat`, `stat_goal`, and `stat_format` columns.

Reply:
Updated the migration in latest to drop these columns.